docs/strategy-customization.md

```diff
@@ -235,7 +235,7 @@ def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFram
     Short-trades need to be supported by your exchange and market configuration!
     Please make sure to set [`can_short`]() appropriately on your strategy if you intend to short.
 
-    ```python
+```python
```

Comment:
I know markdown lint tools will flag this - but it's intentionally indented - as that will move it into the (collapsed) "note" box once you render the documentation